developers/cameras.md

```diff
@@ -0,0 +1,235 @@
+			Interval: Discrete 0.033s (30.000 fps)
+			Interval: Discrete 0.040s (25.000 fps)
+			Interval: Discrete 0.067s (15.000 fps)
+		Size: Discrete 352x288
+			Interval: Discrete 0.033s (30.000 fps)
+			Interval: Discrete 0.040s (25.000 fps)
+			Interval: Discrete 0.067s (15.000 fps)
+		Size: Discrete 320x240
+			Interval: Discrete 0.033s (30.000 fps)
+			Interval: Discrete 0.040s (25.000 fps)
+			Interval: Discrete 0.067s (15.000 fps)
+		Size: Discrete 1920x1080
+			Interval: Discrete 0.033s (30.000 fps)
+			Interval: Discrete 0.040s (25.000 fps)
+			Interval: Discrete 0.067s (15.000 fps)
+
+    pi@raspberrypi:~ $ v4l2-ctl --list-formats-ext --device=/dev/video2
+    ioctl: VIDIOC_ENUM_FMT
+	Index       : 0
+	Type        : Video Capture
+	Pixel Format: 'YU12'
+	Name        : Planar YUV 4:2:0
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 1
+	Type        : Video Capture
+	Pixel Format: 'YUYV'
+	Name        : YUYV 4:2:2
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 2
+	Type        : Video Capture
+	Pixel Format: 'RGB3'
+	Name        : 24-bit RGB 8-8-8
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 3
+	Type        : Video Capture
+	Pixel Format: 'JPEG' (compressed)
+	Name        : JFIF JPEG
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 4
+	Type        : Video Capture
+	Pixel Format: 'H264' (compressed)
+	Name        : H.264
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 5
+	Type        : Video Capture
+	Pixel Format: 'MJPG' (compressed)
+	Name        : Motion-JPEG
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 6
+	Type        : Video Capture
+	Pixel Format: 'YVYU'
+	Name        : YVYU 4:2:2
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 7
+	Type        : Video Capture
+	Pixel Format: 'VYUY'
+	Name        : VYUY 4:2:2
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 8
+	Type        : Video Capture
+	Pixel Format: 'UYVY'
+	Name        : UYVY 4:2:2
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 9
+	Type        : Video Capture
+	Pixel Format: 'NV12'
+	Name        : Y/CbCr 4:2:0
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 10
+	Type        : Video Capture
+	Pixel Format: 'BGR3'
+	Name        : 24-bit BGR 8-8-8
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 11
+	Type        : Video Capture
+	Pixel Format: 'YV12'
+	Name        : Planar YVU 4:2:0
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 12
+	Type        : Video Capture
+	Pixel Format: 'NV21'
+	Name        : Y/CrCb 4:2:0
+		Size: Stepwise 32x32 - 2592x1944 with step 2/2
+
+	Index       : 13
+	Type        : Video Capture
+	Pixel Format: 'BGR4'
+	Name        : 32-bit BGRA/X 8-8-8-8
+    Size: Stepwise 32x32 - 2592x1944 with step 2/2
+    ```
+4. Now, after we did check that both devices can supply H.264 streams, it's possible to go in
```

Comment:
The only step necessary is to go to /camera page, and see if there is a video device available. All of the above steps are done programatically before displaying the video dev as an option.

Also, there is no need for _both_ devices to have H.264, only one.